### PR TITLE
Task nxbt 3436 remove chart dependencies

### PR DESCRIPTION
**Pull request by ataillefer**

### CODE REVIEW

File: ci/helm/preview/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,16 @@
 apiVersion: v1
```
Comment (kevinleturc) [inline ci/helm/preview/Chart.yaml:1]: Shouldn't we need to update the apiVersion to have the dependencies in this file?

File: ci/helm/preview/values.yaml [hunk 2]
```diff
@@ -20,48 +20,41 @@ cleanup:
...
-      persistence:
-        enabled: $PERSISTENCE
+      size: "100Mi"
+  nodeSelector:
+    team: platform
+  tolerations:
+  - effect: NoSchedule
+    key: team
+    operator: Equal
+    value: platform
```
Comment (kevinleturc) [inline ci/helm/preview/values.yaml:60]: There's missing a new line at the end of the file.

File: ci/helm/values-elasticsearch.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    resource: service
+    team: platform
+    usage: "$USAGE"
+tolerations:
+  - key: team
+    operator: Equal
+    value: platform
+    effect: NoSchedule
+nodeSelector:
+  team: platform
```
Comment (kevinleturc) [inline ci/helm/values-elasticsearch.yaml:32]: There's missing a new line at the end of the file.

**Review by kevinleturc — APPROVED** [review]
Great works 👍. Just a couple of cosmetic things.

File: ci/helm/preview/Chart.yaml (hunk 1 above)
Comment (ataillefer) [reply ci/helm/preview/Chart.yaml:1]: Nope, its' resolved client side when using Helm 3.

**Review by kevinleturc — APPROVED** [review]

**Review by kevinleturc — DISMISSED** [review]

**Review by troger — DISMISSED** [review]

**Review by kevinleturc — APPROVED** [review]

**Comment by kevinleturc** [comment]
💪

**Review by troger — APPROVED** [review]

**Pull request merged**